### ISSUE TITLE
Why are the inference results different from the results you posted?

### ISSUE THREAD

**Issue by Asamoun** [showlab/MotionDirector#32]
Hello author, I used the weights you provided for intervention. The seed is the same as the one you set. Why are the results I got different from the results you published? I would be grateful if you could answer my doubts. .
![image](https://github.com/showlab/MotionDirector/assets/122540267/49ad2695-bb86-45b8-a4e2-3e670f73388d)

**Comment by Asamoun** [comment]
I'm very glad that you can reply to my message. I tried changing the torch version, but the result is still the same. The following is the environment I configured. I hope you can take a look:
```
# packages in environment at D:\anaconda\envs\motion1:
# Name                    Version                   Build  Channel
absl-py                   2.1.0                    pypi_0    pypi
accelerate                0.18.0                   pypi_0    pypi
antlr4-python3-runtime    4.9.3                    pypi_0    pypi
appdirs                   1.4.4                    pypi_0    pypi
attrs                     23.2.0                   pypi_0    pypi
ca-certificates           2024.3.11            haa95532_0    defaults
certifi                   2024.2.2                 pypi_0    pypi
cffi                      1.16.0                   pypi_0    pypi
charset-normalizer        3.3.2                    pypi_0    pypi
click                     8.1.7                    pypi_0    pypi
colorama                  0.4.6                    pypi_0    pypi
contourpy                 1.1.1                    pypi_0    pypi
cycler                    0.12.1                   pypi_0    pypi
decord                    0.6.0                    pypi_0    pypi
deepspeed                 0.3.16                   pypi_0    pypi
diffusers                 0.18.0                   pypi_0    pypi
docker-pycreds            0.4.0                    pypi_0    pypi
einops                    0.7.0                    pypi_0    pypi
filelock                  3.13.3                   pypi_0    pypi
fire                      0.6.0                    pypi_0    pypi
flatbuffers               24.3.25                  pypi_0    pypi
fonttools                 4.51.0                   pypi_0    pypi
fsspec                    2024.3.1                 pypi_0    pypi
ftfy                      6.2.0                    pypi_0    pypi
git                       2.44.0               h57928b3_0    defaults
gitdb                     4.0.11                   pypi_0    pypi
gitpython                 3.1.43                   pypi_0    pypi
huggingface-hub           0.16.4                   pypi_0    pypi
idna                      3.6                      pypi_0    pypi
imageio                   2.33.0                   pypi_0    pypi
imageio-ffmpeg            0.4.9                    pypi_0    pypi
importlib-metadata        7.1.0                    pypi_0    pypi
importlib-resources       6.4.0                    pypi_0    pypi
jax                       0.4.13                   pypi_0    pypi
jinja2                    3.1.3                    pypi_0    pypi
kiwisolver                1.4.5                    pypi_0    pypi
libffi                    3.4.4                hd77b12b_0    defaults
libsqlite                 3.45.2               hcfcfb64_0    defaults
lora-diffusion            0.1.7                    pypi_0    pypi
loralib                   0.1.0                    pypi_0    pypi
markupsafe                2.1.5                    pypi_0    pypi
matplotlib                3.7.5                    pypi_0    pypi
mediapipe                 0.10.11                  pypi_0    pypi
ml-dtypes                 0.2.0                    pypi_0    pypi
mpmath                    1.3.0                    pypi_0    pypi
mypy-extensions           1.0.0                    pypi_0    pypi
networkx                  3.1                      pypi_0    pypi
ninja                     1.11.1.1                 pypi_0    pypi
numpy                     1.24.0                   pypi_0    pypi
omegaconf                 2.3.0                    pypi_0    pypi
opencv-contrib-python     4.9.0.80                 pypi_0    pypi
opencv-python             4.8.0.74                 pypi_0    pypi
openssl                   3.0.13               h2bbff1b_0    defaults
opt-einsum                3.3.0                    pypi_0    pypi
packaging                 24.0                     pypi_0    pypi
pandas                    2.0.3                    pypi_0    pypi
pillow                    10.3.0                   pypi_0    pypi
pip                       24.0               pyhd8ed1ab_0    defaults
protobuf                  3.20.3                   pypi_0    pypi
psutil                    5.9.8                    pypi_0    pypi
pycparser                 2.22                     pypi_0    pypi
pyparsing                 3.1.2                    pypi_0    pypi
pyre-extensions           0.0.29                   pypi_0    pypi
python                    3.8.19               h1aa4202_0    defaults
python-dateutil           2.9.0.post0              pypi_0    pypi
pytz                      2024.1                   pypi_0    pypi
pyyaml                    6.0.1                    pypi_0    pypi
regex                     2023.12.25               pypi_0    pypi
requests                  2.31.0                   pypi_0    pypi
safetensors               0.4.2                    pypi_0    pypi
scipy                     1.10.1                   pypi_0    pypi
sentry-sdk                1.44.1                   pypi_0    pypi
setproctitle              1.3.3                    pypi_0    pypi
setuptools                65.6.3             pyhd8ed1ab_0    defaults
six                       1.16.0                   pypi_0    pypi
smmap                     5.0.1                    pypi_0    pypi
sounddevice               0.4.6                    pypi_0    pypi
sqlite                    3.45.2               hcfcfb64_0    defaults
sympy                     1.12                     pypi_0    pypi
tensorboardx              1.8                      pypi_0    pypi
termcolor                 2.4.0                    pypi_0    pypi
tokenizers                0.13.3                   pypi_0    pypi
torch                     2.0.1+cu118              pypi_0    pypi
torchaudio                2.0.2+cu118              pypi_0    pypi
torchvision               0.15.2+cu118             pypi_0    pypi
tqdm                      4.65.0                   pypi_0    pypi
transformers              4.27.4                   pypi_0    pypi
typing-extensions         4.11.0                   pypi_0    pypi
typing-inspect            0.9.0                    pypi_0    pypi
tzdata                    2024.1                   pypi_0    pypi
ucrt                      10.0.22621.0         h57928b3_0    defaults
urllib3                   2.2.1                    pypi_0    pypi
vc                        14.3                hcf57466_18    defaults
vc14_runtime              14.38.33130         h82b7239_18    defaults
vs2015_runtime            14.38.33130         hcb4865c_18    defaults
wandb                     0.16.6                   pypi_0    pypi
wcwidth                   0.2.13                   pypi_0    pypi
wheel                     0.41.2           py38haa95532_0    defaults
xformers                  0.0.20                   pypi_0    pypi
zipp                      3.18.1                   pypi_0    pypi
```
This is the command I used.
`python MotionDirector_inference.py --model ./models/zeroscope_v2_576w  --prompt "A panda is riding a bicycle in a garden." --checkpoint_folder ./outputs2/train/riding_bicycle/ --checkpoint_index 300 --noise_prior 0. --seed 2178639`
![image](https://github.com/showlab/MotionDirector/assets/122540267/7698c20d-1eab-4863-8e09-fbac5bfe172f)
Here, "A panda is riding a bicycle in a garden." still doesn't work very well. Looking forward to your answer, I will be grateful if you can solve my problem

**Comment by Kenneth-Wong** [comment]
+1. "A panda is riding a bicycle in a garden." does not work weel too.

**Comment by LiuHuijie6410** [comment]
+1，I cannot reproduce results as beautiful as the author's.Have you solved this problem?

**Comment by ruizhaocv** [comment]
Sorry for the late reply. I checked the results you provided and found that only the result of the "panda riding a bicycle" is abnormal while other results are the same as the results I posted.
After carefully checking the results, I found that I confused the random seed used for different version models. I found that the seed 2178639 is used by the newer model in the hugging space https://huggingface.co/spaces/ruizhaocv/MotionDirector.
In the demo, input the text prompt with the same seed, and I get the same results as follows. I recommend you to go to the huggingface space to have a try.
![image](https://github.com/user-attachments/assets/268e26ce-e890-4116-91cc-0e0a79152950)
